cds_ils/migrator/utils.py

```diff
@@ -223,3 +232,30 @@ def add_title_from_conference_info(json_data):
             json_data["title"] = conference_info_title
 
     return json_data
+
+
+def get_item_info(record):
+    """Get dict out of item info field."""
+    # DO NOT use at home
```

Comment:
Can you add a more meaningful comment? :)
what's inside `item_info`? No try/except?